backend/typescript/rest/activityRoutes.ts

```diff
@@ -0,0 +1,102 @@
+import { Router } from "express";
+import { activityRequestDtoValidator } from "../middlewares/validators/activityValidators";
+import ActivityService from "../services/implementations/activityService";
+import {
+  ActivityResponseDTO,
+  IActivityService,
+} from "../services/interfaces/activityService";
+import { getErrorMessage, NotFoundError } from "../utilities/errorUtils";
+import { sendResponseByMimeType } from "../utilities/responseUtil";
+
+const activityRouter: Router = Router();
+
+const activityService: IActivityService = new ActivityService();
+
+/* Create Activity */
+activityRouter.post("/", activityRequestDtoValidator, async (req, res) => {
+  try {
+    const { body } = req;
+    const newActivity = await activityService.createActivity({
+      activityName: body.activityName,
+    });
+    res.status(201).json(newActivity);
+  } catch (e: unknown) {
+    if (e instanceof NotFoundError) {
+      res.status(404).send(getErrorMessage(e));
+    } else {
+      res.status(500).send("Internal server error occured.");
```

Comment:
let's wait until jerry's pr is in and then change this to use his constant: `INTERNAL_SERVER_ERROR_MESSAGE`